syntax = "proto3";
package appmesh;
option go_package = "./;main";

message Response {
  string uuid = 1;
  string request_uri = 2;
  string http_body_msg_type = 3;
  map<string, string> headers = 4;
  int32 http_status = 5;
  bytes http_body = 6;
}
